### DISCUSSION TITLE
Welcome to charts.css Discussions!

### DISCUSSION THREAD

**Discussion by ramiy** [ChartsCSS/charts.css#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by redas24** [comment]
Studen self-taught HTML5 CSS3 and JavaScript with networking background I will implement to my first projects  your library looks like suit for,me
